.github/workflows/test_helper.yml

```diff
@@ -129,8 +129,10 @@ jobs:
         run: |
           echo "path filters with changed files: ${{ inputs.path_filter }}"
           if [[ -z "${{ inputs.path_filter }}" ]]; then
+            echo "No files changed"
```

Comment:
Can we actually make this more specific? Maybe

"No path filters provided, running all pytest tests"

Could use some language other than "path filter". Maybe even "No file changes provided" - just to specify that this case occurs when the detect file change step isn't even run (and then passed in to this job in workflow call)